pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -628,9 +628,9 @@ message Image {
     // ID of the image.
     optional string id = 1;
     // Other names by which this image is known.
-    repeated string repo_tags = 2;
+    repeated string names = 2;
```

Comment:
I am asking because the definition could be ambiguous in some cases, and it'd be better to clarify this in the comment. `RepoTags` clearly would not contain the digest, but I am not sure if it's still so straightforward with`Names`. For example, the image `Names` included in the node status [may include both tags and digests](https://github.com/kubernetes/kubernetes/blob/906cb1ce705774722a954ee06f03210d52f3bcbf/pkg/kubelet/kubelet_node_status.go#L540).